src/Athena/Views/Home/Login_Page.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿
+@{
+    ViewData["Title"] = "Login_Page";
+}
+
+<h2>Login_Page</h2>
+
+<div class="input-field col s6">
+    <input id="User_Name" type="text" class="validate">
+    <label for="User_Name">User Name</label>
+</div>
+<div class="input-field col s6">
+    <input id="Password" type="text" class="validate">
+    <label for="Password">Password</label>
+</div>
+<a href="Main_Page" style="color:white" class="btn">Login</a>
+<a href="Home" style="color:white" class="btn">Cancel</a>
```

Comment:
Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.